public/locales/es/home.json

```diff
@@ -1,5 +1,11 @@
 {
     "constructionNotice": "¡Importante! Todavía estamos construyendo este sitio de web. Regresa en abril del 2021.",
     "noActiveSensor": "¡Haz clic en un sensor en el mapa para ver el Índice de la calidad de aire (Air Quality Index, AQI) aquí!",
-    "dataVizBox": "Cuadro de visualización de datos"
+    "dataVizBox": "Cuadro de visualización de datos",
+    "expandMap": "Expandir para ver el mapa de sensores",
```

Comment:
Would we want to conjugate these as singular second person informal commands or leave infinitive @hillaryrguez